packages/components/src/ActionButton.tsx

```diff
@@ -113,11 +106,18 @@ export const ActionButton = ({
   hoverColor,
   ...props
 }: ActionButtonProps<keyof React.ReactHTML>): JSX.Element => {
+  // depending on the props, we can conditionally apply a class
+  const hoverColorClass =
```

Comment:
The outline hover is not working anymore. To check that, you can try to create a new key. The "Import Existing Key" should have a black hover. Regarding this piece of  code, maybe it could be placed as part of the tailwind-variants definitions (https://www.tailwind-variants.org/ for more info). Maybe you could use the compound variants section: https://www.tailwind-variants.org/docs/variants#compound-variants

Reply:
It turns out to be much easier. I just had to add this to variants: 
```css
 hoverColor: {
      primary: "to-yellow",
      secondary: "to-cyan",
      black: "to-black",
},
  ```
  The outlined button was fixed as well